test/showcase-echo-client/src/v1beta1/echo_client.ts

```diff
@@ -1,4 +1,4 @@
-// Copyright 2022 Google LLC
```

Comment:
I'm assuming this file does not have any changes after being generated, right?  (it should not)

Same nit, please run `npx gts fix` to make it formatted properly. It will also reduce the number of differences :)

Reply:
Ran `npx gts fix`. It does look like it has some changes - things that may have just evolved since the last time it was generated? Nothing super fundamental, but I'm changing the copyright year now